packages/ogc/src/common/index.ts

```diff
@@ -0,0 +1,16 @@
+export { default as bboxToString } from "./bbox-to-string";
```

Comment:
Not a huge fan of having one file per function like this. How about merging them into a few files by topics? Some of them also don't seem to be ogc specific?